rita/src/client.rs

```diff
@@ -162,7 +162,12 @@ fn main() {
             .resource("/make_payment", |r| {
                 r.method(Method::POST).with2(make_payments)
             })
-            .resource("/hello", |r| r.method(Method::POST).with2(hello_response))
+            .resource("/hello", |r| {
+                r.method(Method::POST).with2(hello_response_old)
+            })
+            .resource("/hello", |r| {
+                r.method(Method::POST).with2(hello_response_new)
+            })
```

Comment:
Just how hacky is this? Is it something like a fallback, where when one request fails Actix tries the other?

Reply:
yeah, it tries the handlers in order, (they take different json payloads, and its not a "match" (but also not a fail) in actix's eyes if the payload fails to deserialize)

Reply:
Okay, so I understood correctly that this isn't a good practice? would you mind marking this as "not a good practice" in a comment?